services/ai-navigator-app/0.1.0/ai-navigator-app.yaml

```diff
@@ -0,0 +1,52 @@
+# TODO: Should these be separate files?
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ai-navigator-app-config
+  namespace: kommander
+data:
+  CHATBOT_URL: "https://chatbot.api.d2iq.com/v1/chat"
+  AUTHORIZER_URL: "https://auth.api.d2iq.com"
+  LICENSE_NAMESPACE: "kommander" # TODO: use kommander namspace, is this hardcoded ok?
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ai-navigator-app
+  namespace: kommander # TODO: use kommander namspace, is this hardcoded ok?
+  labels:
+    app: ai-navigator-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ai-navigator-app
+  template:
+    metadata:
+      labels:
+        app: ai-navigator-app
+    spec:
```

Comment:
there’s a few things we should set on this deployment like cpu/memory limits, securityContext (run as non-root, etc). but we can handle those later on.

Reply:
You are right - is there a good way to find good limit/request values other than just guessing?

Reply:
the ideal way is to observe the utilization in a running cluster while the app is under some sort of load and estimate based on those values. we have some grafana dashboards builtin that you can use

Reply:
I found this in another application definition:

```
        securityContext:
          allowPrivilegeEscalation: false
          capabilities:
            drop:
            - ALL
          readOnlyRootFilesystem: true
          runAsNonRoot: true
          seccompProfile:
            type: RuntimeDefault
```

Are these the same properties we would need to set for `securityContext` for this application?

Reply:
Security context added! :)